torch/csrc/jit/passes/to_batch.cpp

```diff
@@ -525,11 +526,11 @@ void ToBatch::toBatch(Block* block, Block* res_block) {
 }
 
 std::shared_ptr<Graph> to_batch_graph(std::shared_ptr<Graph>& graph){
-  // std::cout<<graph->toString()<<std::endl;
   std::shared_ptr<Graph> res_graph = std::make_shared<Graph>(graph->scope_root());
   ToBatch to_batch;
   to_batch.toBatch(graph->block(), res_graph->block());
-  // std::cout<<res_graph->toString()<<std::endl;
+
+  EliminateDeadCode(res_graph);
```

Comment:
What happened here?

Reply:
The inlining that this pass does introduced extraneous world tokens. They are harmless affected the readability of the graph, so I added a DCE pass to clean things up.